libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/AdasisConfigPathLevelOptions.kt

```diff
@@ -0,0 +1,71 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Path level options
+ *
+ * @param stub Stub message options
+ * @param segment Segment message options
+ * @param profileShort Profile short message options
+ * @param profileLong Profile long message options
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class AdasisConfigPathLevelOptions(
```

Comment:
internal constructor + builder
Otherwise it'll be more difficult to extend the API.
But probably also not critical for the snapshot.